batchflow/models/tf/nn/train.py

```diff
@@ -0,0 +1,97 @@
+""" Helpers for training """
+from math import pi
+
+import tensorflow as tf
+from tensorflow.math import sin, asin, floor # pylint: disable=import-error
+
+
+def piecewise_constant(global_step, *args, **kwargs):
+    """ Constant learning rate decay (uses global_step param instead of x) """
+    return tf.train.piecewise_constant(global_step, *args, **kwargs)
+
+def cyclic_learning_rate(learning_rate, global_step, max_lr=0.1, step_size=10,
+                         mode='tri', name='CyclicLearningRate'):
+    """ This function varies the learning rate between the
+    minimum (learning_rate) and the maximum (max_lr).
+    It returns the decayed learning rate.
+
+    Leslie N. Smith "`Cyclical Learning Rates
+    for Training Neural Networks <https://arxiv.org/abs/1506.01186>`_"
+
+    Parameters
+    ----------
+    learning_rate: float or tf.Tensor
+        the minimum learning rate boundary
```

Comment:
Use this [guide](https://numpydoc.readthedocs.io/en/latest/format.html) in the docstring

Reply:
Should I use the  'References' section to refer to the article and the 'Notes' section to show how functions are computed?